test/system/001-basic.bats

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+#
+# Simplest set of podman tests. If any of these fail, we have serious problems.
+#
```

Comment:
Is your thinking that we should execute this one in a fail-fast mode?  IIRC the default is all tests run no matter pass/fail.  This might be a good idea, conserving resources and developer time waiting in cases where all tests are simply going to fail anyway.  We can handle this on the automation-side, like:

1. Run just this test, fail on non-zero exit
2. Run all tests, fail on non-zero exit

Reply:
Good question. BATS doesn't (AFAICT) support a fail-fast mode, but I suppose we could define a convention that tests named `00*` are a run-first and fail-early subset. I'll document that but it must of course be left as an exercise for the caller.

Reply:
Yes we're drinking the same cool-aide.  Either way, having anything called "basic" fail straight off the starting gate, is a helpful sign to check the environmental-basics.